tests/plugins/test_orchestrate_build.py

```diff
@@ -448,7 +488,8 @@ def mock_wait_for_build_to_finish(build_name):
                 'build_kwargs': make_worker_build_kwargs(),
                 'osbs_client_config': str(tmpdir),
                 'max_cluster_fails': 2,
-                'unreachable_cluster_retry_delay': .1
+                'unreachable_cluster_retry_delay': .1,
+                'goarch': {'x86_64': 'amd64', 'ppc64le': 'ppc64'},
```

Comment:
Can we also remove `ppc64le` item to ensure the code is not requiring it? Same for other cases.